docs/references/faq.md

```diff
@@ -135,3 +135,147 @@ out_batch = pa.Table.from_arrays([new_data, batch.column('ColumnToKeepAsIs')], s
+    \"requests==2.24.0\", PyPAC==0.15.0\"
+-   **CLI**: \"typer==0.3.1\", \"doit==0.33.1\", \"pydantic==1.8.2\",
+    \"packaging==20.4\", \"requests==2.24.0\", \"xmltodict==0.12.0\"
+
+
+## _"Why another Python SDK?"_
+
+The original [Python Engine
+SDK](https://help.alteryx.com/developer-help/python-engine-sdk) is complex, runs in-process, and can be slow. The [AYX
+Python SDK](https://help.alteryx.com/developer-help/ayx-python-sdk) design solves these issues. It runs
+out-of-process using gRPC to communicate. It also simplifies the
+development process of new tools to 4 function calls, abstracting away a
+lot of the previous SDK function calls.
+
+By running the plugin out-of-process, we are able to efficiently manage
+multiple plugins and are not bound by the restrictions of the C++
+`boost` library.
+
+
+## _"Python Engine SDK and AYX Python SDK. What's the difference?"_
+
+[AYX Python SDK](https://help.alteryx.com/developer-help/ayx-python-sdk-v2) is a new SDK product. It improves upon
+the original Python Engine SDK by providing a consistent development
+experience moving forward. While the Python Engine SDK was designed to
+use the original engine, the new AYX Python SDK leverages the brand new [AMP
+Engine](https://help.alteryx.com/20223/designer/alteryx-amp-engine).
+
+
+## _"Are there any changes to packaging between the Python Engine SDK and AYX Python SDK?"_
+
+In the new SDK, we switched from using packaged up virtual environments
+to python zip apps. We believe this update provides an overall positive
+change because now tools with different environments work on the same
+tool canvas (no virtual environment collisions).
+
+Please note that this might change how you packaged up, distributed, and
+tested your plugins.
+
+
+## _"Do I need Alteryx Designer to build and test a plugin?"_
+
+Alteryx Designer is not required to build a custom plugin. However,
+Designer is required to test a plugin and make sure that it works
+correctly.
+
+
+## _"What IDE should I use?"_
+
+We recommend [Visual Studio Code](https://code.visualstudio.com/) or [PyCharm](https://www.jetbrains.com/pycharm/).
+
+
+## _"What's the purpose of the AYX Plugin CLI?"_
+
+The AYX Plugin CLI provides a single mechanism to guide you through the
+entire SDK development process, from scaffolding to packaging. It lets
+you create tools quickly, familiarize yourself with the SDKs, and it
+reduces the potential for error.
+
+The CLI facilitates the process of custom tool creation for any SDK.
+*Note that at this time it only supports Python.*\
+
+
+## _"What's the difference between sdk-workspace-init and create-ayx-plugin commands?"_
+
+-   `sdk-workspace-init`: This command initializes a workspace directory
+    --- think of this as a project playground where all your custom
+    plugins exist. The workspace directory will be setup with a
+    pre-defined folder structure which enables the rest of the CLI
+    commands to properly function.
```

Comment:
TW review:

`sdk-workspace-init`: This command initializes a workspace directory
    --- think of this as a project playground where all your custom
    plugins exist. The workspace directory is set up with a
    pre-defined folder structure, which enables the rest of the CLI
    commands to properly function.

Reply:
fixed